docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+volumes:
+  node_modules:
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile.localdev
+    extra_hosts:
+      - 'host.docker.internal:host-gateway'
+    command: poetry run python manage.py runserver 0.0.0.0:${PORT:-8000}
```

Comment:
Issue: If we always override the command here, why not just put it in the dockerfile?